app/views/instruments/show.html.erb

```diff
@@ -3,6 +3,8 @@
 
```

Comment:
Here we would iterate over the photos and show them with the cl_image_tag maybe as a carousel.